setup.py

```diff
@@ -14,7 +14,7 @@
       setup_requires=['setuptools_scm', 'setuptools_scm_git_archive'],
       zip_safe=False,
       packages=['Ska', 'Ska.Sun', 'Ska.Sun.tests'],
-      package_data={},
+      package_data={'Ska.Sun': ['data/*fits.gz']},
```

Comment:
I never trust myself with packaging updates like this. Best to do a quick functional install test. I usually do:
```
python setup.py install --user
$ cd ~
$ ipython
import Ska.Sun
Ska.Sun.test()
exit()
$ rm -rf ~/.local  # assuming you keep this clean like I do.
```

Reply:
And are you saying in the edited description that you've done the install test?

Reply:
OK, well I've done it too so it can stay.